src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -212,6 +207,18 @@ public String getResourceURI(IResource resource) {
 		}
 	}
 	
+	private RequestBody getPayload(String method, IResource payload) {
```

Comment:
resolved my own question.  Open issue https://issues.jboss.org/browse/OSJC-249 that resolves